.circleci/config.yml

```diff
@@ -91,6 +91,9 @@ commands:
       - run:
           name: Install js dependencies
           command: pnpm install
+      - run:
+          name: Build charts vendor
+          command: node ./packages/x-charts-vendor/scripts/build.js
```

Comment:
Should we provide the package built already instead of having to build it in the pipelines and docs?

Reply:
This would create an inconsistency in the release:

- For not the `@mui/x-charts-vendor` is not published so no preview could work
- The preview would used the last released version of the `@mui/x-charts-vendor` which is not practical if you need to bump one of the d3-lib